README.md

```diff
@@ -63,7 +63,14 @@ The [pre-provisioned dashboard](grafana/definitions/mlt.json) demonstrates a [RE
 
 [Data links](https://grafana.com/docs/grafana/latest/panels-visualizations/configure-data-links/), [exemplars](https://grafana.com/docs/grafana-cloud/data-configuration/traces/exemplars/), and logs are utilized to allow jumping from the dashboard to a Grafana Explore page to observe traces, metrics, and logs in more detail.
 
-The following is a brief explanation of each of the most important provided components.
+The following sections are a brief explanation of each of the most important provided components.
+
+### Using the OpenTelemetry Collector
+
+You can swap out the Grafana Agent for the OpenTelemetry collector using an alternative configuration.
+
+Please see the 'OpenTelemetry Collector' section below to use this environment instead.
```

Comment:
```suggestion
Red the 'OpenTelemetry Collector' section below to use this environment instead.
```

Reply:
Done. And actually, seven other instances!